rsciio/tests/test_topspin.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2025 The HyperSpy developers
+#
+# This file is part of RosettaSciIO.
+#
+# RosettaSciIO is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# RosettaSciIO is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RosettaSciIO. If not, see <https://www.gnu.org/licenses/#GPL>.
+
+import xml.etree.cElementTree as ET
+from contextlib import redirect_stdout
+from io import StringIO
+from pathlib import Path
+
+import h5py
```

Comment:
```suggestion
h5py = pytest.importorskip("h5py")
```
to be able to run the test suite when h5py is not installed, for example for the CI build with minimum dependencies.